draft-ietf-quic-recovery.md

```diff
@@ -289,8 +289,14 @@ be subtracted from the RTT as long as the result is larger than the min_rtt.
 If the result is smaller than the min_rtt, the RTT should be used, but the
 ack delay field should be ignored.
 
-Like TCP, QUIC calculates both smoothed RTT and RTT variance similar to those
-specified in {{?RFC6298}}.
+A sender calculates both smoothed RTT and RTT variance similar to those
+specified in {{?RFC6298}}, see {{on-ack-received}}.
+
+A sender will commonly receive ACK frames that acknowledge larger packet numbers
```

Comment:
Please add text on why this is relevant.  Suggestion "that increase largest acked and therefore generate an RTT sample."

Reply:
I've taken your suggestion but rephrased to avoid a run-on sentence.